catalog/USGS/USGS_NLCD_RELEASES_2019_REL_RCMAP_V5_TRENDS_YEAR.jsonnet

```diff
@@ -0,0 +1,534 @@
+local id = 'USGS/NLCD_RELEASES/2019_REL/RCMAP/V5/TRENDS/YEAR';
+local subdir = 'USGS';
+local version = '5';
+
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+local units = import 'units.libsonnet';
+
+local license = spdx.cc0_1_0;
+
+local basename = std.strReplace(id, '/', '_');
+local base_filename = basename + '.json';
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+
+// TODO(schwehr): Move these into units.libsonnet.
+local units_counts = '1 (count)';
+local units_year = 'a';
+local units_centipercent_per_year = 'c%/a';  // % change/year × 100
+local units_slope = 'c%/a';  // % change/year × 100
+local units_confidence = '1 (confidence)';

```

Comment:
I think it's better to add a new unit explicitly called "p-value" because "confidence" sometimes can be expressed in different ways

Reply:
these aren't needed, see updated jsonnet - should be resolved I believe